lib/mixlib/install/generator/bourne/scripts/platform_detection.sh

```diff
@@ -55,19 +55,14 @@ elif test -f "/etc/redhat-release"; then
     platform="el"
   fi
 
+# detect amazon linux 2013/2014 which lack /etc/os-release
```

Comment:
/etc/system-release is some fedora-ism, though, not just amazon linux.

Reply:
yeah i think /etc/redhat-release -> /etc/system-release in fedora at some point, but then systemd's /etc/os-release came along.  amazon uses /etc/system-release because they're a "fedoraish" distro which has not yet gone to /etc/os-release, but that can't necessarily be relied upon to uniquely detect amazon.

Reply:
(or it looks like they do have /etc/os-release...   anyway point stands that none of these files are unique to amazon..)

Reply:
2015+ they switched to os-release. We already had the os-release logic so this way we're just going to continue the same old logic we had, but if we get back amzn from os-release we'll convert it to amazon. That will be a nice future proof way to get the version number without brittle regexing of the release name. For 2013/2014 we'll fall back to that crummy old regex.